uptime/monitor-setup/checks.json

```diff
@@ -620,14 +972,73 @@
         "schedule": []
       },
       "monitoring_service_type": "TRANSACTION",
+      "is_paused": true,
+      "is_under_maintenance": false,
+      "state_is_up": false,
+      "state_changed_at": "2023-03-15T15:39:56.399435Z",
+      "msp_protocol": "",
+      "msp_interval": 5,
+      "msp_address": "registry.developer.gov.bc.ca",
+      "msp_port": null,
+      "msp_username": "",
+      "msp_proxy": "",
+      "msp_dns_server": "",
+      "msp_dns_record_type": "",
+      "msp_status_code": "",
+      "msp_send_string": "",
+      "msp_expect_string": "",
+      "msp_expect_string_type": "STRING",
+      "msp_encryption": "",
+      "msp_threshold": 60,
+      "msp_headers": "",
+      "msp_script": "[{\"step_def\":\"C_OPEN_URL\",\"values\":{\"url\":\"https://registry.developer.gov.bc.ca\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\"button\",\"text\":\"Login\",\"is_regex\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".css-1jhnpo2\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT\",\"values\":{\"element\":\"#password\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#user\",\"text\":\"psta\",\"typing_delay\":\"\"}},{\"step_def\":\"C_FILL_FIELD\",\"values\":{\"element\":\"#password\",\"text\":\"p1aTf0rmS3rV!c3sT3aM\",\"typing_delay\":\"\"}},{\"step_def\":\"C_MOUSE_CLICK\",\"values\":{\"element\":\".login-form-action > .btn-primary\",\"button\":\"left\",\"click_count\":\"1\",\"skip_navigation\":\"\"}},{\"step_def\":\"C_WAIT_FOR_ELEMENT_TEXT\",\"values\":{\"element\":\".css-lgt0d3 .css-vurnku > .css-vurnku\",\"text\":\"Git documentation for Platform Services Registry\",\"is_regex\":\"\"}}]",
```

Comment:
same here!

Reply:
you might need to request to update the IDIR account secret just in case!

Reply:
Also the N8N API key + the url completely opens up N8N